apps/hashdotdev/src/pages/roadmap.page.tsx

```diff
@@ -0,0 +1,268 @@
+import { Box, Container, Typography, typographyClasses } from "@mui/material";
+import { FunctionComponent, ReactNode } from "react";
+
+import { AnchorTag } from "../components/anchor-tag";
+import { Button } from "../components/button";
+import { FaIcon } from "../components/icons/fa-icon";
+import { Link } from "../components/link";
+import { PageLayout } from "../components/page-layout";
+import { NextPageWithLayout } from "../util/next-types";
+import { BlueStylishDivider } from "./blog/shared/blue-styled-divider";
+import { TechnologyTree } from "./roadmap/technology-tree";
+import { useCases } from "./roadmap/use-cases";
+
+const headingLinks: { label: string; href: string; icon: ReactNode }[] = [
+  {
+    label: "Use Cases",
+    href: "#use-cases",
+    icon: <FaIcon name="ballot-check" type="regular" />,
+  },
+  {
+    label: "Technology Tree",
+    href: "#technology-tree",
+    icon: <FaIcon name="diagram-project" type="regular" />,
+  },
+  {
+    label: "Get Involved",
+    href: "#get-involved",
+    icon: <FaIcon name="teddy-bear" type="regular" />,
+  },
+];
+
+const Head: FunctionComponent = () => (
+  <Container sx={{ marginBottom: 10 }}>
+    <Box>
+      <Typography variant="hashHeading4" component="h2" mb={3}>
+        Where is HASH at
+      </Typography>
+      <Typography variant="hashLargeTitle" mb={5} sx={{ lineHeight: 1.1 }}>
+        Roadmap
+      </Typography>
+      <BlueStylishDivider mb={5} />
+      <Box display="flex">
+        <Box width={{ xs: 1, md: 725 }}>
+          <Typography mb={2} sx={{ lineHeight: 1.5 }}>
+            Here you’ll find the features we’re intending to build, information
+            around the order in which we’ll be addressing them, as well as
+            functional and technical specifications.
+          </Typography>
+          <Typography sx={{ lineHeight: 1.5 }}>
+            We’ve mapped these features to use-cases, so if you’re interested in
+            using HASH for a particular thing, you can follow along (or even
+            contribute!)
+          </Typography>
+        </Box>
+        <Box
+          marginLeft={16}
+          sx={{
+            marginLeft: 16,
+            display: {
+              lg: "block",
+              xs: "none",
+            },
+          }}
+        >
+          <Typography marginBottom={1}>
+            <strong>Quick-links</strong>
```

Comment:
@nonparibus for view
```suggestion
            <strong>Quick links</strong>
```

Reply:
This copy is also from the Figma, @nonparibus would you like me to make this change?